### PR TITLE
Add a simple resource via a PUT 

### PR DESCRIPTION
**Pull request by whikloj**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -111,6 +111,13 @@
       <scope>test</scope>
       <type>test-jar</type>
     </dependency>
+        <dependency>
```
Comment (awoods) [inline pom.xml]: please format

File: src/test/java/org/fcrepo/integration/auth/webac/WebACRecipesIT.java [hunk 2]
```diff
@@ -65,31 +70,36 @@ public WebACRecipesIT() {
...
-
         logger.debug("setup complete");
     }
 
     @Test
-    public void test() {
+    public void scenario1() throws Exception {
+        logger.info("Running scenario1");
+        final String objA = getRandomPid();
+        final HttpPut method = new HttpPut(serverAddress + "/" + objA);
```
Comment (awoods) [inline src/test/java/org/fcrepo/integration/auth/webac/WebACRecipesIT.java]: Please use: super.putObjMethod()

File: src/test/java/org/fcrepo/integration/auth/webac/WebACRecipesIT.java [hunk 3]
```diff
@@ -65,31 +70,36 @@ public WebACRecipesIT() {
...
+        final FileEntity acl =
+                new FileEntity(new File(classLoader.getResource("acls/01/acl.ttl").getFile()));
+        setAuth(method, "fedoraAdmin");
+        method.setHeader("Content-type", "text/turtle");
+        method.setEntity(acl);
+        final HttpResponse response = client.execute(method);
```
Comment (awoods) [inline src/test/java/org/fcrepo/integration/auth/webac/WebACRecipesIT.java]: Please use: 
```bash
try (final CloseableHttpResponse response = execute(method)) {
         ...
        }
```

File: src/test/java/org/fcrepo/integration/auth/webac/WebACRecipesIT.java [hunk 4]
```diff
@@ -65,31 +70,36 @@ public WebACRecipesIT() {
...
+        final String content = EntityUtils.toString(response.getEntity());
+        final int status = response.getStatusLine().getStatusCode();
```
Comment (awoods) [inline src/test/java/org/fcrepo/integration/auth/webac/WebACRecipesIT.java]: use `super.getStatus()`

File: src/test/java/org/fcrepo/integration/auth/webac/WebACRecipesIT.java [hunk 5]
```diff
@@ -56,6 +59,8 @@
 
     protected static CloseableHttpClient client;
 
+    protected final ClassLoader classLoader = getClass().getClassLoader();
```
Comment (awoods) [inline src/test/java/org/fcrepo/integration/auth/webac/WebACRecipesIT.java]: Why not `private`?

File: src/test/java/org/fcrepo/integration/auth/webac/WebACRecipesIT.java [hunk 6]
```diff
@@ -56,6 +59,8 @@
 
     protected static CloseableHttpClient client;
```
Comment (awoods) [inline src/test/java/org/fcrepo/integration/auth/webac/WebACRecipesIT.java]: Why not `private`?

File: src/test/java/org/fcrepo/integration/auth/webac/WebACRecipesIT.java [hunk 7]
```diff
@@ -65,31 +70,37 @@ public WebACRecipesIT() {
...
+        logger.info("Running scenario1");
+        final String objA = getRandomPid();
+        final HttpPut method = super.putObjMethod(serverAddress + "/" + objA);
+        final FileEntity acl =
+                new FileEntity(new File(classLoader.getResource("acls/01/acl.ttl").getFile()));
+        setAuth(method, "fedoraAdmin");
+        method.setHeader("Content-type", "text/turtle");
+        method.setEntity(acl);
+        try (final CloseableHttpResponse response = super.execute(method)) {
+            final String content = EntityUtils.toString(response.getEntity());
```
Comment (awoods) [inline src/test/java/org/fcrepo/integration/auth/webac/WebACRecipesIT.java]: The body of the response does not help much, and in the name of less/cleaner code, I would suggest removing it and changing the assertion below to:
```bash
assertEquals(CREATED.getStatusCode(), status);
```

File: src/test/java/org/fcrepo/integration/auth/webac/WebACRecipesIT.java (hunk 6 above)
Comment (whikloj) [inline src/test/java/org/fcrepo/integration/auth/webac/WebACRecipesIT.java]: Do I even need this if I'm using the super.execute()?

**Comment by awoods** [comment]
Resolved with: https://github.com/fcrepo4-labs/fcrepo-module-auth-webac/commit/47528aca1cab21d5a38774e1299c59c2cb802134

**Pull request closed without merging**